src/app/Component/ChatScreen/chatscreen.css

```diff
@@ -176,30 +176,6 @@ p{
     box-shadow: 0 0 15px #e4e0e6c0 ; 
 }
 
-textarea {
-    font-family: inherit;
-    font-size: 100%;
-    vertical-align: baseline;
-    border: 0;
-    outline: 0;
-    color: #fff;
-  }
-
-  textarea {
```

Comment:
Please don't use same css classnames or elements.It will face a issue while bundling the single js file.Please try to use unique class names